backend/tests/seeder/test_seeder_user.py

```diff
@@ -0,0 +1,41 @@
+from seeder.user import (
+    seed_users,
+    interactive_seeder,
+    User,
+    User_Properties
+)
+from sqlmodel import Session, select
+
+
+def test_seed_users(session: Session):
+    # Test seeding users
+    seed_users(session=session, count=10)
+    result = session.exec(select(User)).all()
+    assert len(result) > 10
+    result_up = session.exec(select(User_Properties)).all()
+    assert len(result_up) > 0
+
+
+def test_interactive_seeder(session: Session, monkeypatch):
+    # Simulate user input for interactive seeder
+    user_input = [
+        "123456789",  # Phone number
```

Comment:
If this simulates a real HTTP call, this phone number will have the leading `+`.

Reply:
Hi @kjkoster , 
This is only for seeding data in development.